CHANGELOG.asciidoc

```diff
@@ -188,6 +188,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 
 *Filebeat*
 
+- Add lowercase processor to iis access pipeline {issue}7534[7534]
```

Comment:
Could you changes to something like `Change field "iis.access.hostname" to always be lower case`. Otherwise users reading this don't really know what changed in the user facing part.

Reply:
That makes sense, thank you.